src/GraphQLParser/ParserContext.cs

```diff
@@ -240,8 +240,8 @@ private GraphQLArgument ParseArgument()
 
         private List<GraphQLInputValueDefinition>? ParseArgumentDefs()
         {
-            return Peek(TokenKind.PAREN_L)
-                ? Many(TokenKind.PAREN_L, (ref ParserContext context) => context.ParseInputValueDef(), TokenKind.PAREN_R)
+            return Peek(TokenKind.PAREN_L)
+                ? Many(TokenKind.PAREN_L, (ref ParserContext context) => context.ParseInputValueDef(), TokenKind.PAREN_R)
```

Comment:
Hereafter, I see changes in diff view without actual changes. Could you please leave only significant changes in order to simplify and speed up the review?

Reply:
Looks like Rider updated whitespaces, didn't see them on the changelist. Will update

Reply:
Cleaned up the changes. It would be good to reformat the document though, it looks like the problem was with inconsistent newlines

Reply:
Let's do reformatting in separate PR. I will add [.editorconfig](https://github.com/graphql-dotnet/graphql-dotnet/blob/master/.editorconfig) some time later.